jdl/converters/types.ts

```diff
@@ -1,21 +1,26 @@
 import { RelationshipType, RelationshipSide } from '../basic-types/relationships.js';
 
-export type Field = {
+export type JSONField = {
   fieldName: string;
   fieldType: string;
   options?: Record<string, boolean | string | number>;
 } & Record<string, any>;
 
-export type Relationship = {
+export type JSONRelationship = {
   relationshipSide?: RelationshipSide;
   relationshipName: string;
   relationshipType: RelationshipType;
   otherEntityName: string;
   options?: Record<string, boolean | string | number>;
 } & Record<string, any>;
 
-export type Entity = {
+export type JSONEntity = {
```

Comment:
I'm a bit confused with this one: what's the difference with `generators/base-application/types/entity.d.ts`? is it relevant that these `converters/types` exists?

Reply:
`generators/base-application/types/entity.d.ts` have calculated values and supported annotations.
They are used in generators.
In jdl and json only base data and annotations of type `Record<any, any>` exists.